README.md

```diff
@@ -5,3 +5,8 @@
 This repository contains:
 - Specific JSON-LD contexts and NGSI-LD sample payloads used in our current projects.
 - A discussion on the use of [unit code](UnitCodes.md) in NGSI-LD payloads and transformation from senML units.
+
+## Contribution guidelines
+- New files and new directories names must be `camelCase`
+- Intermediary directories such as `jsonld-contexts` and `ngsild-payloads` are `kebab-case`
+- For compounds contexts files, use `camelCase` for the base name and apply `-compound` as a suffix (i.e: specialContext-compoun.jsonld)
```

Comment:
```suggestion
- For compounds contexts files, use `camelCase` for the base name and apply `-compound` as a suffix (i.e: specialContext-compound.jsonld)
```

Reply:
Other than the typo already mentioned, looks good to me.